src/UI/class-settings-page.php

```diff
@@ -672,18 +847,84 @@ public function validate_options( array $input ): array {
 				);
 			}
 		}
-		// these can't be null, if somebody accidentally deselected them just reset to default.
-		if ( ! isset( $input['track_post_types'] ) ) {
-			$input['track_post_types'] = array( 'post' );
-		}
-		if ( ! isset( $input['track_page_types'] ) ) {
-			$input['track_page_types'] = array( 'page' );
+
+		$input['api_secret'] = sanitize_text_field( $input['api_secret'] );
+
+		if ( ! empty( $input['metadata_secret'] ) ) {
+			if ( strlen( $input['metadata_secret'] ) !== 10 ) {
```

Comment:
This `10` seems like a magic number that should be made into a class constant to better self-document it.

Reply:
I see that this is just moved code, not new code.